tests/baselines/reference/computedPropertyNames48_ES5.types

```diff
@@ -17,6 +17,7 @@ enum E { x }
 
 var a: any;
 >a : any
+>  : ^^^

```

Comment:
What happened that updated the types baselines like this?

Reply:
This is a goofy quirk of the types baselines; in `typeWriter.ts`:

```ts
// Distinguish `errorType`s from `any`s; but only if the file has no errors.
```

So this types file changed because `computedPropertyNames48_ES5.errors.txt` was emitted.

Reply:
(Honestly, I wonder if it's worth the special case; I noticed it separately while working on something else)